<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configurationWindow</class>
 <widget class="QDialog" name="configurationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="PlayerLabel">
      <property name="text">
       <string>Players</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="PlayerSpinBox">
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="OkButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="CancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
